proposal/slash-separator.md

```diff
@@ -0,0 +1,415 @@
+
+[old rgb]: ../accepted/color.md#rgb-and-rgba
+
+* If `$channels` is a [special variable string][], return a plain CSS function
+  string with the name `"rgb"` and the argument `$channels`.
+
+  [special variable string]: ../spec/functions/color.md#special-variable-string
+
+* If `$channels` is an unbracketed slash-separated list:
+
+  * If `$channels` doesn't have exactly two elements, throw an error. Otherwise,
+    let `rgb` be the first element and `alpha` the second element.
+
+  * If either `rgb` or `alpha` is a special variable string, return a plain CSS
+    function string with the name `"rgb"` and the argument `$channels`.
+
+  * If `rgb` is not an unbracketed space-separated list, throw an error.
+
+  * If `rgb` has more than three elements, throw an error.
+
+  * If `rgb` has fewer than three elements:
+
+    * If any element of `rgb` is a [special variable string][], return a
+      plain CSS function string with the name `"rgb"` and the argument
+      `$channels`.
+
+    * Otherwise, throw an error.
+
+  * Let `red`, `green`, and `blue` be the three elements of `rgb`.
+
+  * Call `rgb()` with `red`, `green`, `blue`, and `alpha` as arguments and
+    return the result.
+
+* Otherwise, proceed with the existing definition of the function.
+
+> This ensures that calling (for example) `rgb(0 0 100% / 80%)` will continue to
+> work when `/` is parsed as a slash separator. It's important to define this
+> runtime behavior in phase 1 so that users manually constructing
+> slash-separated lists can use them as expected.
+
+### `hsl()` Function
+
+This proposal modifies [the existing behavior][old hsl] of the `hsl($channels)`
+overload to be the following:
+
+[old hsl]: ../accepted/color.md#hsl-and-hsla
+
+* If `$channels` is a [special variable string][], return a plain CSS function
+  string with the name `"hsl"` and the argument `$channels`.
+
+* If `$channels` is an unbracketed slash-separated list:
+
+  * If `$channels` doesn't have exactly two elements, throw an error. Otherwise,
+    let `hsl` be the first element and `alpha` the second element.
+
+  * If either `hsl` or `alpha` is a special variable string, return a plain CSS
+    function string with the name `"hsl"` and the argument `$channels`.
+
+  * If `hsl` is not an unbracketed space-separated list, throw an error.
+
+  * If `hsl` has more than three elements, throw an error.
+
+  * If `hsl` has fewer than three elements:
+
+    * If any element of `hsl` is a [special variable string][], return a
+      plain CSS function string with the name `"hsl"` and the argument
+      `$channels`.
+
+    * Otherwise, throw an error.
+
+  * Let `hue`, `saturation`, and `lightness` be the three elements of `hsl`.
+
+  * Call `hsl()` with `hue`, `saturation`, `lightness`, and `alpha` as arguments
+    and return the result.
+
+* Otherwise, proceed with the existing definition of the function.
+
+## Deprecation Process
+
+The deprecation process will be divided into three phases:
```

Comment:
It could be helpful to include examples of the recommended way for end users to write division and `/` separated lists in their Sass files during each phase.

Reply:
Good idea, done.